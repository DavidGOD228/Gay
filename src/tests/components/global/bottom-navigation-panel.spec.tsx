import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import BottomNavigationPanel from '../../../components/global/bottom-navigation/bottom-navigation-panel'
import UiBlockHorizontalEdges from '../../../components/ui/block/horizontal-edges'
import UiBlockVerticalCenter from '../../../components/ui/block/vertical-center'
import {Text} from 'react-native'

it('renders correctly', () => {
  const tree = renderer.create(
    <BottomNavigationPanel>
      <UiBlockVerticalCenter>
        <UiBlockHorizontalEdges>
          <Text> The answer to life is 42</Text>
        </UiBlockHorizontalEdges>
      </UiBlockVerticalCenter>
    </BottomNavigationPanel>
  )
  expect(tree)
    .toMatchSnapshot()
})
