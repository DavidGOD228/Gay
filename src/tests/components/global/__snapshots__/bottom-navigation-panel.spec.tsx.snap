// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#1D4C83",
      "height": 65,
      "paddingLeft": 15,
      "paddingRight": 15,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
      >
         The answer to life is 42
      </Text>
    </View>
  </View>
</View>
`;
